CHANGES

```diff
@@ -15,6 +49,11 @@ Incompatible changes
   has been longly used as default of sphinx-quickstart.
 * LaTeX: Move message resources to ``sphinxmessage.sty``
 * LaTeX: Stop using ``\captions<lang>`` macro for some labels
+* LaTeX: Greek letters in text are not escaped to math mode mark-up, and they
+  will use the text font not the math font. If (and only if) the document
+  contains such Greek Unicode letters *and* the :confval:`latex_engine` is
+  ``'pdflatex'`` then the :confval:`latex_elements`.\ ``'fontenc'`` key
+  **must** be used to declare usage of the ``LGR`` font encoding.
```

Comment:
Initially I added LGR and usage of packages `textalpha` and `substitutefont` to the default pdflatex setting so that no document would need any change for transition to Sphinx 2.0. Unfortunately the `textalpha` is on Ubuntu xenial available via `texlive-lang-greek` which wasn't previous dependency.

So after some hesitation I decided on the contrary to not change anything to the default `'fontenc'` but this means that if a document had a Greek Unicode in text, which previously was escaped to math mark-up, now, what will happens is that project author must explicitely use `latex_elements` `'fontenc'`  key to add usage of ``LGR`` encoding. And the TeX installation might need additional LaTeX packages (listed above).